docs/design/block-design.md

```diff
@@ -4,14 +4,85 @@ The following is a light guide to designing a new block with recommendations and
 
 ## Best Practices
 
-- Blocks should have a simple label for the Inserter. Keep it as short as possible.
-- Blocks should have an identifying icon, ideally using a single color. Try to avoid using the same icon used by an existing block. The core block icons are based on [Material Design Icons](https://material.io/tools/icons/). Look to that icon set, or to [Dashicons](https://developer.wordpress.org/resource/dashicons/) for style inspiration.
-- Blocks should have a instructive placeholder state when they’re first inserted. If the block includes a text input, provide placeholder text. If your block holds media, include buttons for uploading files and accessing media libraries, as well as drop-zones for drag-and-drop. 
-- When unselected, your block should preview its content as closely to the front-end output as possible.
-- When selected, your block may surface additional options like input fields or buttons to configure the block directly, if those are necessary for basic operation.
-- Every block should include a description in the “Block” tab of the Settings sidebar. The description should explain as clearly as possible what your block does. Keep it to a single sentence.
-- The “Block” tab of the Settings Sidebar can contain additional block options and configuration, but keep in mind that a user might dismiss the sidebar and never use it. Do not put critical options there.
-- Check how your block looks, feels, and works on all sorts of devices and screen sizes.
+**Blocks**
+
+Blocks should have a simple label for the Block Library. Keep it as short as possible.
+
+Blocks should have an identifying icon, ideally using a single color. Try to avoid using the same icon used by an existing block. The core block icons are based on [Material Design Icons](https://material.io/tools/icons/). Look to that icon set, or to [Dashicons](https://developer.wordpress.org/resource/dashicons/) for style inspiration.
+
+Do:
+
+![blocks-do](./blocks-do.png)
+
+Don't:
+
+![blocks-dont](./blocks-dont.png)
+
+---
+
+**Placeholders**
+
+Blocks should have a instructive placeholder state when they’re first inserted. If the block includes a text input, provide placeholder text. If your block holds media, include buttons for uploading files and accessing media libraries, as well as drop-zones for drag-and-drop. 
```

Comment:
I'd clarify this with "If your block requires a user to configure some options before you can display it, you should provide an instructive placeholder state."

Not sure about the placeholder text suggestion — we should check in with a11y to see what the best practice is there.

Reply:
cc @tofumatt :)